Automattic/newspack-theme
# Update screenshot so logo uses the correct primary colour

### All Submissions:






### Changes proposed in this Pull Request:

The theme's screenshot should use the correct logo/colour, which is #3366ff.

### How to test the changes in this Pull Request:

1. Go to Appearance > Themes. Do you see the Newspack Theme screenshot with the logo?
2. Switch to this branch.
3. Refresh Appearance > Themes. You should now see the same logo but with a different blue.

### Other information:
